src/Administration/Resources/app/administration/src/app/asyncComponent/media/sw-media-preview-v2/sw-media-preview-v2.spec.js

```diff
@@ -226,4 +226,49 @@ describe('src/app/asyncComponent/media/sw-media-preview-v2', () => {
 
         expect(wrapper.vm.previewUrl).toEqual(wrapper.vm.source);
     });
+
+    it('should return an empty string if trueSource has no thumbnails', async () => {
+        const wrapper = await createWrapper();
+        await wrapper.setData({
+            trueSource: { thumbnails: [] },
+        });
+
+        expect(wrapper.vm.sourceSet).toBe('');
+    });
+
+    it('should return a formatted string of thumbnail sources', async () => {
+        const wrapper = await createWrapper();
+        await wrapper.setData({
+            trueSource: {
+                thumbnails: [
+                    { url: 'https://example.com/image1.jpg', width: 200 },
+                    { url: 'https://example.com/image2.jpg', width: 400 },
+                ],
+            },
+        });
+
+        expect(wrapper.vm.sourceSet).toBe(
+            'https://example.com/image1.jpg 200w, https://example.com/image2.jpg 400w'
+        );
+    });
+
+    it('should return an empty string if trueSource is a file', async () => {
+        const wrapper = await createWrapper();
+        const trueSource = new File([''], 'example.jpg', { type: 'image/jpg' });
+        trueSource.thumbnails = [];
+
+        await wrapper.setData({ trueSource });
+
+        expect(wrapper.vm.sourceSet).toBe('');
+    });
+
+    it('should return an empty string if trueSource is a URL', async () => {
```

Comment:
because you have to adjust something again anyway:
```suggestion
    it('should return an empty string if trueSource is an URL', async () => {
```

Reply:
Please don't. URL starts with an "Yu" sound, which is why "a URL" is correct.

Reply:
I checked this right in the moment. sorry 😅

Reply:
no worries 😄